02.extensions/zooport/02.troubleshooting/02.faq/docs.md

```diff
@@ -0,0 +1,15 @@
+---
+title: FAQ
+taxonomy:
+    category: docs
+---
+
+* **Does the standard ZOO Import/Export support ZOOlanders elements?"** - No, this is currently unsupported. 
+
+* **Is an option "Import from installed software : Joomla / K2" supported?** - CSV import is supported, if the software allows to export the data in CSV. JSON import supports only JSON files generated by ZOOport, due to the specific format. 
```

Comment:
We can remove the question about "Import from installed software : Joomla / K2" option as this block is removed now on import page